spec/controllers/people_controller_spec.rb

```diff
@@ -28,8 +28,8 @@
     let(:census_employee_id) {employee_roles[0].census_employee_id}
 
     let(:email_attributes) { {"0"=>{"kind"=>"home", "address"=>"test@example.com"}}}
-    let(:addresses_attributes) { {"0"=>{"kind"=>"home", "address_1"=>"address1", "address_2"=>"", "city"=>"city1", "state"=>"DC", "zip"=>"22211"},
-        "1"=>{"kind"=>"home", "address_1"=>"address1", "address_2"=>"", "city"=>"city1", "state"=>"DC", "zip"=>"22211"},
+    let(:addresses_attributes) { {"0"=>{"kind"=>"home", "address_1"=>"address1", "address_2"=>"", "city"=>"city1", "state"=>"DC", "zip"=>"22211", "_id"=> person.addresses[0].to_s},
+        "1"=>{"kind"=>"home", "address_1"=>"address1", "address_2"=>"", "city"=>"city1", "state"=>"DC", "zip"=>"22211", "_id"=> person.addresses[0].to_s},
```

Comment:
Do different addresses with the same id?

Reply:
It's a typo. It need to be a [1]. I will push the changes. Thanks